### PR TITLE
DevEx: Use yaml instead of a custom documentation format for entity docs

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -260,5 +260,8 @@
   },
   "resolutions": {
     "minimist": "^1.2.5"
+  },
+  "dependencies": {
+    "json2yaml": "^1.1.0"
```
Comment (rachaelparris) [inline package.json]: This should be under `devDependencies` maybe?

File: shared/src/tools/generateMarkdownSchema.js [hunk 2]
```diff
@@ -351,4 +194,3 @@ generateMarkdownSchema(Case, 'Case');
 generateMarkdownSchema(CaseDeadline, 'CaseDeadline');
 generateMarkdownSchema(DocketRecord, 'DocketRecord');
 generateMarkdownSchema(Document, 'Document');
-generateMarkdownSchema(Statistic, 'Statistic');
```
Comment (rachaelparris) [inline shared/src/tools/generateMarkdownSchema.js]: Did you remove this one on purpose?

Comment (ericsorenson) [reply shared/src/tools/generateMarkdownSchema.js]: I didn't! I'll add it back.

File: package.json (hunk 1 above)
Comment (ericsorenson) [reply package.json]: Yes! Good catch!

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**